CMakeLists.txt

```diff
@@ -90,86 +85,56 @@ option(IREE_ENABLE_EMITC "Enables MLIR EmitC dependencies." ${IREE_BUILD_COMPILE
 # Target and backend configuration
 #-------------------------------------------------------------------------------
 
-# List of all runtime HAL drivers included in the project:
-set(IREE_ALL_HAL_DRIVERS
-  CUDA
-  Dylib
-  Dylib_Sync
-  VMVX
-  VMVX_Sync
-  Vulkan
-)
-
-if(IREE_HAL_DRIVERS_TO_BUILD STREQUAL "default")
-  set(IREE_HAL_DRIVERS_TO_BUILD ${IREE_ALL_HAL_DRIVERS})
+option(IREE_HAL_DRIVER_DEFAULT "Sets the default value for all runtime HAL drivers" ON)
```

Comment:
worth making this plural (DEFAULTS) or something - "IREE_HAL_DRIVER_DEFAULT" makes me think it's picking _a_ default driver, not that it's using the set of default drivers.

Reply:
SG, changed to plural.